browser/blueprints.py

```diff
@@ -334,3 +337,11 @@ def shortcut(filename):
         title=title,
         filename=filename,
         settings=settings)
+
+
+# Factory for Edit objects
+def get_edit(file_, output_bucket, rgb):
+    if is_npz_file(file_.filename):
+        return ZStackEdit(file_, output_bucket, rgb)
+    elif is_trk_file(file_.filename):
+        return TrackEdit(file_, output_bucket)  # don't use RGB mode with track files
```

Comment:
Nice! May as well put the comment as a docstring:

```suggestion
def get_edit(file_, output_bucket, rgb):
    """Factory for Edit objects"""
    if is_npz_file(file_.filename):
        return ZStackEdit(file_, output_bucket, rgb)
    elif is_trk_file(file_.filename):
        return TrackEdit(file_, output_bucket)  # don't use RGB mode with track files
```

Also, what should this function do if the file is neither a track of zstack file?

Reply:
As the factory takes a CalibanFile object as an argument, the CalibanFile object would have already thrown an error if it were not passed a track or zstack file. Still, a BaseEdit could be an appropriate default case.